src/httpEnvelope.js

```diff
@@ -0,0 +1,39 @@
+var _ = require( "lodash" );
+
+function HttpEnvelope( req, res ) {
+	this.transport = "http";
+	this.context = req.context;
+	this.cookies = req.cookies;
+	this.data = req.body || {};
```

Comment:
@Prestaul that's an interesting thought. I haven't worried about it up to this point but maybe that is a mistake? I can't presently think of a reason it would cause issues since the only consumer of that value is our handler and I don't know of anything else that would use the raw express request body. Cloning will slow down processing every request and use more memory, so we need a good reason to do that. That actually makes me think of a place in hyped where I'm cloning that should probably be revisited.